OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs

```diff
@@ -20,28 +22,37 @@ namespace OpenRA.Mods.Common.Scripting
 	[ScriptPropertyGroup("Ability")]
 	public class CaptureProperties : ScriptActorProperties
 	{
-		readonly CapturesInfo normalInfo;
+		readonly Captures[] capturesTraits;
 		readonly ExternalCapturesInfo externalInfo;
 
 		public CaptureProperties(ScriptContext context, Actor self)
 			: base(context, self)
 		{
-			normalInfo = Self.Info.TraitInfoOrDefault<CapturesInfo>();
+			capturesTraits = Self.TraitsImplementing<Captures>().ToArray();
 			externalInfo = Self.Info.TraitInfoOrDefault<ExternalCapturesInfo>();
 		}
 
 		[Desc("Captures the target actor.")]
 		public void Capture(Actor target)
 		{
-			var normalCapturable = target.Info.TraitInfoOrDefault<CapturableInfo>();
-			var externalCapturable = target.Info.TraitInfoOrDefault<ExternalCapturableInfo>();
+		var normalCapturable = target.Info.TraitInfoOrDefault<CapturableInfo>();
```

Comment:
Style nit: there is no reason to qualify this as "normal" capturing.  Just call it `capturable`.